### PR TITLE
Fix listener logic, bump cody commit

### PR DESCRIPTION
**Pull request by pkukielka**
## Test plan

1. Run Document Code action in the IDE
2. Cancel it before accept lens appear

### CODE REVIEW

**Review by mkondratek — APPROVED** [review]

File: src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt [hunk 1]
```diff
@@ -398,7 +397,14 @@ abstract class FixupSession(
   override fun dispose() {
     if (project.isDisposed) return
     performedActions.forEach { it.dispose() }
-    editor.document.removeDocumentListener(documentListener)
+
+    try {
+      editor.document.removeDocumentListener(documentListener)
+    } catch (e: Throwable) {
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: it will not help

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: the error is already logger as internal error by `logger.error()`

Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: Yea you are right. We need to look at it tomorrow then.

Comment (abeatrix) [reply src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: is it possible to check if `documentListener` is valid before `editor.document.removeDocumentListener`? or does the error come from somewhere else?

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: the error is already reported to the IDE within `removeDocumentListener` - we cannot catch it 😕 

I merged this one and I am going to do the release. I will try to fix this issue later.

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt:403]: a few moments later: https://github.com/sourcegraph/jetbrains/pull/1621

**Pull request merged**